variables.tf

```diff
@@ -124,3 +124,7 @@ variable "base_url" {
   description = "base URL for iron service"
   default     = ""
 }
+variable "iron_credentials" {
+  type        = string
```

Comment:
Does this work? Or should it be `map(any)`. If it works as a string, then it's fine!